dashboards/network/vpc_detail.sp

```diff
@@ -0,0 +1,628 @@
+dashboard "ibm_is_vpc_detail" {
+
+  title = "IBM VPC Detail"
+  documentation = file("./dashboards/network/docs/vpc_detail.md")
+
+  tags = merge(local.network_common_tags, {
+    type = "Detail"
+  })
+
+  input "vpc_crn" {
+    title = "Select a VPC:"
+    sql   = query.ibm_is_vpc_input.sql
+    width = 4
+  }
+
+  container {
+
+    card {
+      width = 2
+      query = query.ibm_is_vpc_num_ips_for_vpc
+      args  = {
+        crn = self.input.vpc_crn.value
+      }
+    }
+
+    card {
+      width = 2
+      query = query.ibm_subnet_count_for_vpc
+      args  = {
+        crn = self.input.vpc_crn.value
+      }
+    }
+
+  }
+
+  container {
+
+    container {
+
+      width = 6
+
+      table {
+        title = "Overview"
+        type  = "line"
+        width = 6
+        query = query.ibm_is_vpc_overview
+        args  = {
+          crn = self.input.vpc_crn.value
+        }
+      }
+
+      table {
+        title = "Tags"
+        width = 6
+        query = query.ibm_is_vpc_tags
+        args  = {
+          crn = self.input.vpc_crn.value
+        }
+      }
+
+    }
+
+    container {
+
+      width = 6
+
+      table {
+        title = "Address Prefixes"
+        query = query.ibm_is_vpc_address_prefixes
+        args  = {
+          crn = self.input.vpc_crn.value
+        }
+      }
+
+    }
+
+  }
+
+  container {
+
+    title = "Subnets"
+
+    chart {
+      title = "Subnets by Zone"
+      type  = "column"
+      width = 4
+      query = query.ibm_is_vpc_subnet_by_zone
+      args = {
+        crn = self.input.vpc_crn.value
+      }
+    }
+
+    table {
+      query = query.ibm_is_vpc_subnets_for_vpc
+      width = 8
+      args = {
+        crn = self.input.vpc_crn.value
+      }
+    }
+
+  }
+
+  container {
+
+    title = "NACLs"
+
+    flow {
+      base  = flow.nacl_flow
+      title = "Inbound NACLs"
+      width = 6
+      query = query.ibm_inbound_nacl_for_vpc_sankey
+      args = {
+        crn = self.input.vpc_crn.value
+      }
+    }
+
+
+    flow {
+      base  = flow.nacl_flow
+      title = "Outbound NACLs"
+      width = 6
+      query = query.ibm_outbound_nacl_for_vpc_sankey
+      args = {
+        crn = self.input.vpc_crn.value
+      }
+    }
+
+  }
+
+
+  table {
+    title = "Cloud Service Endpoint Source Addresses"
+    query = query.ibm_is_vpc_cse_source_ip_addresses
+    args = {
+      crn = self.input.vpc_crn.value
+    }
+  }
+
+
+  table {
+    title = "Default Route Tables"
+    query = query.ibm_is_vpc_default_route_tables
+    args = {
+      crn = self.input.vpc_crn.value
+    }
+  }
+
+  table {
+    title = "Default Security Group"
+    query = query.ibm_is_vpc_default_security_group
+    args = {
+      crn = self.input.vpc_crn.value
+    }
+  }
+
+  table {
+    title = "Default NACL"
```

Comment:
```suggestion
    title = "Default Network ACL"
```

Reply:
Better have a table with all the ACLs shown as in the analysis chart and place it below the Sankey diagram